.goreleaser.yml

```diff
@@ -49,3 +49,30 @@ brews:
         type: optional
     test: |
       system "#{bin}/zero version"
+
+nfpms:
+  -
+    id: zero
+    package_name: zero
+    vendor: Commit.dev
+    homepage: https://www.commit.dev
+    maintainer: commitdev <contact@commit.dev>
+    description: "Allow startup developers to ship to production on day 1."
+    formats:
+      - apk
+      - deb
+      - rpm
+
+    recommends:
+      - awscli
```

Comment:
Are `terraform` and `kubectl` available as dependencies?

Reply:
No, unfortunately. Not for *any* of the linux targets.